coastlines/raster.py

```diff
@@ -38,15 +38,16 @@
 import odc.algo
 import odc.geo.xr
 from datacube.utils.aws import configure_s3_access
-from datacube.utils.cog import write_cog
-from datacube.utils.geometry import CRS, GeoBox, Geometry
+from odc.geo.geom import CRS, Geometry
```

Comment:
Use `odc-geo` functions instead of `datacube` functions (will make upgrading to `datacube==1.9` easier)